tagging/routes/routeGetMostUsedTags.go

```diff
@@ -54,3 +56,35 @@ func routeGetMostUsedTags(r *http.Request, a routes.Arguments) (int, interface{}
 		"mostUsedTags": tagsList,
 	}
 }
+
+func routeGetMostUsedTagsHistory(r *http.Request, a routes.Arguments) (int, interface{}) {
+	logger := jsonlog.LoggerFromContextOrDefault(r.Context())
+	u := a[users.AuthenticatedUser].(users.User)
+	tx := a[db.Transaction].(*sql.Tx)
+	dbRes, err := models.MostUsedTagsHistoryByUser(tx, u.Id)
+	if err != nil {
+		logger.Error("Could not fetch most used tags.", err.Error())
+		return http.StatusInternalServerError, err.Error()
+	}
+
+	if dbRes == nil {
+		return http.StatusInternalServerError, errors.New("no history reports available")
+	}
+	history := make([]map[string]interface{}, 0)
+	for _, mut := range dbRes {
+		tagsList := []string{}
+		err = json.Unmarshal([]byte(mut.Tags), &tagsList)
+		if err != nil {
+			logger.Error("Could not unmarshal most used tags.", err.Error())
+			return http.StatusInternalServerError, err
```

Comment:
Use `continue` so a little error does not block the user to get valid history data.